girder/utility/install.py

```diff
@@ -130,6 +132,41 @@ def _pipeOutputToProgress(proc, progress):
             proc.wait()
 
 
+def getGitVersions():
```

Comment:
To channel @brianhelba here, I think we should make this symbol private.

Reply:
Sure.